### PR TITLE
[KHRGA-84] Updated EP nd_range classes

### PR DESCRIPTION
**Pull request by PetroBondar**
Issue: KHRGA-84

### CODE REVIEW

**Comment by PetroBondar** [comment]
There is also a small typo in the Specs in the [Table 110](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.constructors.ndrange) (brackets instead of comma).

File: source/iface/nd_range.rst [hunk 1]
```diff
@@ -2,70 +2,81 @@
...
-execute a kernel on a set of work items.
-
+  template <int Dimensions = 1>
+  class nd_range;
 
 .. rubric:: Template parameters
 
 ================  ===
-``dimensions``    Number of dimensions
+``Dimensions``    Number of dimensions of ``sycl::nd_range``.
```
Comment (mkinsner) [inline source/iface/nd_range.rst:19]:
```suggestion
``Dimensions``    Number of dimensions in the ``sycl::nd_range``.
```

File: source/iface/nd_range.rst [hunk 2]
```diff
@@ -2,70 +2,81 @@
...
 ================  ===
 
+The ``sycl::nd_range`` defines the iteration domain of both
```
Comment (mkinsner) [inline source/iface/nd_range.rst:22]:
```suggestion
The ``sycl::nd_range`` class defines the iteration domain of both
```

File: source/iface/nd_range.rst [hunk 3]
```diff
@@ -2,70 +2,81 @@
...
+the work-groups and the overall dispatch.
+
+To define this the ``sycl::nd_range`` comprises two
```
Comment (mkinsner) [inline source/iface/nd_range.rst:25]:
```suggestion
A ``sycl::nd_range`` comprises two
```

**Comment by PetroBondar** [comment]
Fixed comments in the https://github.com/KhronosGroup/SYCL_Reference/pull/110/commits/03b26226bf8df4c3dbc7ab0b8564af85dc484a83.

**Review by mkinsner — APPROVED** [review]

**Comment by mkinsner** [comment]
> There is also a small typo in the Specs in the [Table 110](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.constructors.ndrange) (brackets instead of comma).

Thanks!  PR https://github.com/KhronosGroup/SYCL-Docs/pull/505

**Pull request merged**